### PR TITLE
Improvements to Orchestration Key-Concepts Page

### PR DESCRIPTION
**Pull request by amessbee**
Here is a list of changes:
1. Fixed errors in the structure + grammar
2. Removed ICA figure from cosmos (theme looks out of place in our docs)
3. Merged Key Concepts and API pages as they had, somewhat, fluffy content.

[Orchestration Key Concept Page Rendered](https://ms-update-orch-concepts.documentation-7tp.pages.dev/guides/orchestration/getting-started/key-concepts.html)

Refs:
- https://github.com/Agoric/documentation/issues/1179
- https://github.com/Agoric/documentation/issues/1143

### CODE REVIEW

**Review by dckc — COMMENTED** [review]
I defer to @mitdralla and @Jovonni

**Comment by mitdralla** [comment]
- The page (and docs) are really starting to come together. I have some feedback around the main navigation parents/children items. I think we could use more hierarchy to how the children links are organized - I think it is at the point where a category label could be helpful, and even further down the road a mega-menu type layout

image

- For the side nav - "takeaway x:" seems unnecessary, I would just title and label it "Starting a Local Chain" and update the other items.
image

- There is a lot of truncation on the right side nav, where the first words are more generic and the truncated part seems more important - consider cutting those words down so we can see the full title

image

Example: "Creating your Dapp from a Template" -> "Creating from a Template"
Example: "Starting the Dapp Smart Contract" -> "Starting the Smart Contract"

- The page is long and could seem overwhelming for a Getting Started page. Consider adding in the copy up top "Get started in 5 steps" to help set some expectations.

or visit https://classic.yarnpkg.com/en/docs/install for more information -> or visit the official [yarn package](https://classic.yarnpkg.com/en/docs/install)

- I would guide them to the next logical location to hand hold them a bit more like "creating your first smart contract with our hello world example"
image

**Comment by amessbee** [comment]
@mitdralla thanks for your comments. They are very helpful and I or @mujahidkay will create a separate issue based on your feedback. My current PR is about a (key concepts and APIs) page in our new orchestration section. If you can also provide feedback on that it would be great. Below is a direct link to the updated page that needs to be reviewed:

[Orchestration Key Concepts](https://ms-update-orch-concepts.documentation-7tp.pages.dev/guides/orchestration/getting-started/key-concepts.html)

Sorry for the confusion - I should have been clearer when I requested feedback. Looking forward.

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/key-concepts.md:49]: To be consistent with the no-numbering in the previous section, the Orchestrator Interface
```suggestion
### Address Management
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/key-concepts.md:57]:
```suggestion
### Balance Management
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/key-concepts.md:67]:
```suggestion
### Funds Transfer
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/key-concepts.md:82]: Please make this section (ChainHub) consistent with previous two sections with regard to listing different APIs separately. For example I think you can add two sub-headings for registerChain and registerConnection in this section.

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (toliaqat) [inline main/guides/orchestration/getting-started/key-concepts.md:108]: Also, please maket his section consistent by adding APIs subsections. Also currently it seems to be vague and marketing speak. For example, it is not clear to me at first what is ICA after all in the code example bellow. 
Are omniAccount and localAccount ICA objects? If yes, then can you add getAddress, delegate, and transferSteps as API subsections in this ICA section?

**Review by toliaqat — CHANGES_REQUESTED** [review]

**Comment by mitdralla** [comment]
One more item which is minor - there is a missing space: "asAmountconverts a denom amount to an Amount with a brand."  
image

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (amessbee) [reply main/guides/orchestration/getting-started/key-concepts.md:108]: I was thinking of removing this subsection but kept it for now. I don't understand it well enough to be certain, but I believe we do not use the ICA in the code that is externally accessible to the developers. Rather we use the orchestration account that I already explain above.

**Comment by turadg** [comment]
Removing myself. This is DevRel purview.

**Review by toliaqat — APPROVED** [review]
Looks very good. Thanks for updating this page. I am approving from the editorial side. From the technical side, @Jovonni can have a quick look. If there are any minor technical issues, I would request to make subsequent PRs after merging this.

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (mujahidkay) [inline main/guides/orchestration/getting-started/key-concepts.md:97]:
```suggestion
and use of chain and connection information using the following APIs:
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (mujahidkay) [inline main/guides/orchestration/getting-started/key-concepts.md:102]: I was able to follow the flow of these docs up until i read **reincarnation of the vat** especially since we haven't discussed it in orchestration docs before. What exactly is it? I think everything after the first sentence should be additional info presented in a `tip` section

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (mujahidkay) [inline main/guides/orchestration/getting-started/key-concepts.md:101]:
```suggestion
- `registerChain` register a new chain with `chainHub`. The name will override a name in well-known chain names. If a
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (mujahidkay) [inline main/guides/orchestration/getting-started/key-concepts.md:112]:
```suggestion
- `getChainsAndConnection` is used to get chain and connection info given primary and counter chain names.
```

**Review by mujahidkay — APPROVED** [review]
Great work! Please consider squashing the recent commits into an appropriate parent commit

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [reply main/guides/orchestration/getting-started/key-concepts.md]: IIRC "incarnation" represents before & after an upgrade. 

After an upgrade you have a new incarnation. Maybe think of "version" when you see this.

Before an upgrade, the vat may be incarnation # 1, and then after an upgrade, the vat may be incarnation # 2

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [inline main/guides/orchestration/getting-started/key-concepts.md:15]: any reason we are removing this?

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [inline main/guides/orchestration/getting-started/key-concepts.md:46]: not sure its helpful, but I find myself being asked, "what does promise.all do?" more often that I expected when showing this 👀

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [inline main/guides/orchestration/getting-started/key-concepts.md:64]: What is happening under the hood here, is IF we invoke `getBalance` on an remote account, this is actually an Interchain Query (ICQ). We can probably take the opportunity to briefly explain this here if this is the first time they see it. 

Example from orch dapp contract:
```js
const remoteChainBalance = await remoteAccount.getBalance('uosmo');
```

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [reply main/guides/orchestration/getting-started/key-concepts.md:89]: i agree, and take the opportunity to explain each. This is a good set of api functions though

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (mujahidkay) [reply main/guides/orchestration/getting-started/key-concepts.md:10]: I believe we will. This is only the first of at least a few PRs to iron out orch docs

**Comment by amessbee** [comment]
@Jovonni thanks a lot for detailed review and comments. In essence, what we are doing here, everything that is not absolutely necessary for devs and posting it in a separate page: PR #1209. 
We want this page to be magical and really **abstract** out all the details, conveying the message that *if you need to orchestrate it is as simple as calling `getChain`, `makeAccount`, and then `tansfer` for you and Agoric will take care of everything under-the-hood, and BTW if you need to know what's going on under-the-hood, here is an other page with details*. 

WDYT?

File: main/guides/orchestration/getting-started/key-concepts.md
Comment (Jovonni) [reply main/guides/orchestration/getting-started/key-concepts.md:89]: @amessbee  i'd say its good this is prob the only comment to address before merging.

**Review by Jovonni — CHANGES_REQUESTED** [review]
LGTM 🚀 , this is the only thing to address:
https://github.com/Agoric/documentation/pull/1201/files#r1755891367

**Comment by amessbee** [comment]
> LGTM 🚀 , this is the only thing to address: https://github.com/Agoric/documentation/pull/1201/files#r1755891367

@Jovonni this comment by @toliaqat has already been addressed - if you look at the file `chainHub` section already has two subsections. Let me know if there are any further comments.

**Review by Jovonni — APPROVED** [review]

**Pull request merged**
